Classes/Finishers/RedirectFinisher.php

```diff
@@ -55,7 +57,11 @@ public function executeInternal()
 
         $uriParts = parse_url($uri);
         if (!isset($uriParts['scheme']) || $uriParts['scheme'] === '') {
-            $uri = $request->getHttpRequest()->getBaseUri() . $uri;
+            $uri = sprintf(
+                '%s/%s',
+                rtrim($request->getHttpRequest()->getAttribute(Request::ATTRIBUTE_BASE_URI), '/'),
```

Comment:
Any reason why `$request->getHttpRequest()->getBaseUri()` got replaced with `$request->getHttpRequest()->getAttribute(Request::ATTRIBUTE_BASE_URI)`? The former was removed with Flow 6.0, but as Christian mentioned that should not be necessary for that version, so we need to carefully adjust this when upmerging.

Reply:
Not really. I will change it back so it does not break when using neos/flow < 5.1